app/models/concerns/project_search.rb

```diff
@@ -51,6 +52,14 @@ def exact_name
       name
     end
 
+    def extra_searchable_names
+      if platform == "Maven"
+        name.split(":")
```

Comment:
So this will let searches return both group id and artifact id? That sounds reasonable I think

Reply:
yes.